server/routes/_util.js

```diff
@@ -0,0 +1,18 @@
+
+// in express, if you pass query params like this `?param[]=value`
+// then the type of the param is Array and the code may be expecting a string.
+// this convenience function allows either form to be used.
+function arrayParam( param ){
```

Comment:
This is cool. I wonder if we should switch around how we do things in the API. Right now we're very strict. This format and the also comomon `?param=value1&param=value2` form are not allowed.